### PR TITLE
Add more-info JSON example

### LINKED ISSUE #1: Create example JSON file for organization information page

### PR DESCRIPTION
**Pull request by danielabreo**
closes #1

### CODE REVIEW

**Comment by SabrinaSnider** [comment]
> @googlebot I consent

**Comment by ccyang314** [comment]
Updating diffbase to Sabrina's branch to reduce noise in diff.

File: public/data/more-info.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		"name": "XYZ Homeless Shelter",
+		"description": "A description of the non-profit.",
+		"address": "12345 Road Drive",
+		"phone": 1234567890,
+		"website": "www.example.com",
+		"email": "hello@example.com",
+		"items": [{
+			"quality": ["quality check 1", "quality check 2"],
+			"instructions": ["instruction 1", "instruction 2"]
+		}]
```
Comment (ccyang314) [inline public/data/more-info.json:12]: Could you elaborate more on the inner structure of `items`? What is the meaning of  `quality ` and `instructions `?

Comment (ccyang314) [reply public/data/more-info.json:12]: Instead of using `quality check 1`, `quality check 2`, maybe switching to real-world example will also help.

Comment (SabrinaSnider) [reply public/data/more-info.json:12]: I think we will need a 'name' field for each item as well.

**Comment by brucezheng** [comment]
Please add me after all collaborators have approved

**Pull request closed without merging**